src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/StatisticalTableFunction.cs

```diff
@@ -69,38 +71,60 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
                 }
 
                 return false;
-            }
+            }
+
+            var args = callNode.Args.Children.VerifyValue();
+            var fieldType = binding.GetType(args[1]);
+
+            // if a non-numeric feature is enabled and the type is not a number, also check for the specific sub type capability

```

Comment:
nit: comments should start with a capital letter (I thought this was enforced by the linter, I guess not?)